app/server/components/repositories/teams.js

```diff
@@ -17,8 +18,58 @@ getTeam = (teams, prefix, permission) => {
 
 module.exports = {
 
-    edit (username, repoId, team) {
 
+    //checks if a permissive team exists to manage a given repo at specified permission
+    check (repoId, permission) {
+
+        return new Promise((resolve, reject) => {
+
+            debug('checking if team exists for repo [' + repoId + '] at [' + permission + ']');
```

Comment:
nbd, but we can use template strings with babel: https://babeljs.io/docs/learn-es6/#template-strings